content/influxdb/v2.0/reference/glossary.md

```diff
@@ -1055,8 +1055,8 @@ Related entries: [point](#point), [unix timestamp](#unix-timestamp), [RFC3339 ti
 Tokens (or authentication tokens) verify user and organization permissions in InfluxDB.
 There are different types of athentication tokens:
 
-- **Admin token:** grants full read and write access to all resources in **all organizations in InfluxDB OSS 2.x**.
-  _InfluxDB Cloud does not support Admin tokens._
+- **Operator token:** grants full read and write access to all resources in **all organizations in InfluxDB OSS 2.x**.
```

Comment:
Curious if we called these "operators" in the UI/cli/api? For full access to everything across orgs, operator doesn't seem as intuitive as admin